src/test/java/teammates/sqlui/webapi/CreateNotificationActionTest.java

```diff
@@ -0,0 +1,167 @@
+package teammates.sqlui.webapi;
+
+import java.util.UUID;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.NotificationStyle;
+import teammates.common.datatransfer.NotificationTargetUser;
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.Notification;
+import teammates.ui.output.NotificationData;
+import teammates.ui.request.InvalidHttpRequestBodyException;
+import teammates.ui.request.NotificationCreateRequest;
+import teammates.ui.webapi.CreateNotificationAction;
+
+/**
+ * SUT: {@link CreateNotificationAction}.
+ */
+public class CreateNotificationActionTest extends BaseActionTest<CreateNotificationAction> {
+    private static final String GOOGLE_ID = "user-googleId";
+    private static final String TEST_NOTIFICATION = "notification1";
+    NotificationCreateRequest testReq;
+    private final NotificationAttributes testNotificationAttribute = typicalBundle.notifications.get(TEST_NOTIFICATION);
```

Comment:
sorry I missed this one earlier, but same thing as the other PRs, you can use the methods from `src/test/java/teammates/test/BaseTestCase.java`